### DISCUSSION TITLE
Lost coins and transactions

### DISCUSSION THREAD

**Discussion by Mathewskry** [Chia-Network/chia-blockchain#17004] [Support]
Hi  ,three months ago I topped up my wallet with 12.89 coins and when I decided to log in again all the transactions and coins were gone, I'm sure they are there, how can I access them?
Or is there a synchronization problem, both icons are always yellow, what should I do?

**Comment by BrandtH22** [comment]
Hey @Mathewskry , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia) . 

This does sound like a syncing issue to me, the wallet will only be able to sync to the height of the node itself so if that is still syncing then it is expected the wallet will not sync fully. 

One thing you can try just to check the funds is to switch to wallet mode in the settings and restart the chia client (you will need to restart even though the node tab disappears to ensure the node is fully offline and the wallet is connecting to other peers).

To get the node back in sync I would take a look at that node tab to ensure see if it is currently syncing (height is increasing). If not then I would check if you are connected to peers (click on the node icon at the top right to see peers). If it is connected to peers and still not syncing I would check the logs for any errors and also ensure that the system time is correct (I use this site to verify system time: https://time.is/)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
